src/tools/rest-frontend/public/assets/translations/en.json

```diff
@@ -0,0 +1,370 @@
+    "APP.ENTRIES.CREATE.NOTIFICATION.MESSAGE.ENTRY_INVALID_CONFIGURATION_VALUE": "The submitted configuration snippet could not be parsed within the submitted format. If you are sure the format is correct, please try one of the other plugins.",
+    "APP.ENTRIES.CREATE.NOTIFICATION.MESSAGE.ENTRY_INVALID_DESCRIPTION": "The submitted description does not match the requirements.",
+    "APP.ENTRIES.CREATE.NOTIFICATION.MESSAGE.ENTRY_INVALID_ORGANIZATION": "The submitted organization name does not match the requirements.",
+    "APP.ENTRIES.CREATE.NOTIFICATION.MESSAGE.ENTRY_INVALID_SCOPE": "The submitted application scope does not match the requirements",
+    "APP.ENTRIES.CREATE.NOTIFICATION.MESSAGE.ENTRY_INVALID_SLUG": "The submitted slug does not match the requirements.",
+    "APP.ENTRIES.CREATE.NOTIFICATION.MESSAGE.ENTRY_INVALID_TAG": "One or more of the submitted tags does not match the requirements.",
+    "APP.ENTRIES.CREATE.NOTIFICATION.MESSAGE.ENTRY_INVALID_TITLE": "The submitted title does not match the requirements.",
+    "APP.ENTRIES.CREATE.NOTIFICATION.MESSAGE.ENTRY_MISSING_APPLICATION": "You have to submit an application name.",
+    "APP.ENTRIES.CREATE.NOTIFICATION.MESSAGE.ENTRY_MISSING_CONFIGURATION_FORMAT": "You have to submit the format of your snippet.",
+    "APP.ENTRIES.CREATE.NOTIFICATION.MESSAGE.ENTRY_MISSING_CONFIGURATION_VALUE": "You have to submit a configuration snippet.",
+    "APP.ENTRIES.CREATE.NOTIFICATION.MESSAGE.ENTRY_MISSING_DESCRIPTION": "You have to submit a description for the snippet.",
+    "APP.ENTRIES.CREATE.NOTIFICATION.MESSAGE.ENTRY_MISSING_ORGANIZATION": "You have to submit an organization name.",
+    "APP.ENTRIES.CREATE.NOTIFICATION.MESSAGE.ENTRY_MISSING_SCOPE": "You have to submit an application scope.",
+    "APP.ENTRIES.CREATE.NOTIFICATION.MESSAGE.ENTRY_MISSING_SLUG": "You have to submit a slug.",
+    "APP.ENTRIES.CREATE.NOTIFICATION.MESSAGE.ENTRY_MISSING_TITLE": "You have to submit a title for the snippet.",
+    "APP.ENTRIES.CREATE.NOTIFICATION.MESSAGE.NEED_AUTHENTICATION": "You have to be authenticated to perform this action. If the error remains, please refresh the page and login again if necessary.",
+    "APP.ENTRIES.CREATE.NOTIFICATION.MESSAGE.REQUEST_MALFORMED_DATA": "An error occured. Please contact the administrator, if the error persists.",
+    "APP.ENTRIES.CREATE.NOTIFICATION.MESSAGE.REQUEST_UNSUPPORTED_CONTENT_TYPE": "An error occured. Please contact the administrator, if the error persists.",
+    "APP.ENTRIES.CREATE.PANEL.HEADING": "Create new configuration snippet",
+    
+    "APP.ENTRIES.DELETE.BUTTON.ABORT": "No, abort.",
+    "APP.ENTRIES.DELETE.BUTTON.DELETE": "Yes, delete snippet!",
+    "APP.ENTRIES.DELETE.LABEL.BODY": "Do you really want to delete the snippet <b>{{ meta.title }}</b>?",
+    "APP.ENTRIES.DELETE.LABEL.TITLE": "Delete configuration snippet",
+    "APP.ENTRIES.DELETE.NOTIFICATION.HEADER": "Delete configuration snippet",
```

Comment:
Yeah but entirely other scope. If we ever have the idea to change a key in future, it would be very cumbersome to add new ones just to get rid of the multi-usage.